docs/src/appendix/snforge-library/get_call_trace.md

```diff
@@ -0,0 +1,109 @@
+# `get_call_trace`
+
+```rust
+fn get_call_trace() -> CallTrace;
+```
+
+(For whole structure definition, please refer to [`snforge-std` source](https://github.com/foundry-rs/starknet-foundry/tree/v0.16.0/snforge_std))
+
+Gets latest call trace of the test, including the last call made to a contract.
```

Comment:
I think this description is unclear. It should be sth like 
"Gets current call trace of the test up to the last call made to a contract."